sys/dev/ath/if_ath_usb.c

```diff
@@ -457,9 +461,14 @@ ath_usb_attach(device_t self)
 	sc->ops.write = ath_usb_write;
 	sc->ops.write_barrier = ath_usb_write_barrier;
 #endif
-
 	mtx_init(&sc->sc_mtx, device_get_nameunit(self), MTX_NETWORK_LOCK, MTX_DEF);
 
+	// ATH_LOCK_INIT(sc);
```

Comment:
Are these lines needed?

Reply:
It is to investigate mutexes so I will leave it